Packs/TrustwaveFusion/Integrations/TrustwaveFusion/TrustwaveFusion_test.py

```diff
@@ -0,0 +1,22 @@
+"""Base Integration for Cortex XSOAR - Unit Tests file
+
+Pytest Unit Tests: all funcion names must start with "test_"
+
+More details: https://xsoar.pan.dev/docs/integrations/unit-testing
+
+MAKE SURE YOU REVIEW/REPLACE ALL THE COMMENTS MARKED AS "TODO"
+
+You must add at least a Unit Test function for every XSOAR command
+you are implementing with your integration
+"""
+
+import json
+import io
+
+
+def util_load_json(path):
+    with io.open(path, mode='r', encoding='utf-8') as f:
+        return json.loads(f.read())
+
+
+# TODO: ADD HERE unit tests for every command
```

Comment:
please add unittests

Reply:
See [Unit Testing](https://xsoar.pan.dev/docs/integrations/unit-testing) for more details.